app/components/publications/publication/case/mandatees-panel.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @argument {PublicationFlow} publicationFlow (publication-flow,publication-flow.mandatees,publication-flow.mandatees.person)
+ */
+export default class PublicationsPublicationCaseMandateesPanelComponent extends Component {
+  @tracked isOpenLinkModal = false;
+  @tracked mandatees;
+
+  @action
+  openLinkModal() {
+    this.isOpenLinkModal = true;
+  }
+
+  @action
+  closeLinkModal() {
+    this.isOpenLinkModal = false;
+  }
+
+  @action
+  async onLink() {
+    this.isOpenLinkModal = false;
+  }
```

Comment:
Enige bemerking die ik nog heb, is om deze action `didLink` te noemen (naar analogie met `onSave` vs `didSave`) aangezien het linken en opslaan al gebeurd is in de modal component zelf.

Ofwel (en dat heeft mijn voorkeur) het toevoegen van de mandataris en het opslaan van de publication-flow naar hier verhuizen. Op die manier wordt de modal component onafhankelijk van de publication-flow en kan die mogelijk op andere plaatsen ook nog gebruikt worden in de toekomst.

Reply:
Heb dit gedaan.